scripts/zkai_attention_kv_proof_route_selector_gate.py

```
@@ -1871,6 +2077,61 @@ def validate_multihead_quantized_softmax_receipt(summary: Any) -> None:
+    if summary["source_head_count"] != 2 or summary["sequence_length_per_head"] != 16:
+        raise AttentionKvRouteSelectorError("long-sequence fused Softmax shape drift")
+    if summary["lookup_claims"] != 336 or summary["score_rows"] != 336:
+        raise AttentionKvRouteSelectorError("long-sequence fused Softmax lookup drift")
+    if summary["trace_rows"] != 512 or summary["table_rows"] != 9:
+        raise AttentionKvRouteSelectorError("long-sequence fused Softmax trace/table drift")
+    if summary["source_proof_size_bytes"] != 52366 or summary["fused_proof_size_bytes"] != 54234:
+        raise AttentionKvRouteSelectorError("long-sequence fused Softmax proof-size drift")
+    if summary["fused_envelope_size_bytes"] != 1000098:
+        raise AttentionKvRouteSelectorError("long-sequence fused Softmax envelope-size drift")
+    if summary["source_plus_sidecar_raw_proof_bytes"] != 0:
+        raise AttentionKvRouteSelectorError("long-sequence fused Softmax comparator drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)